src/clue/data-colors.scss

```diff
@@ -8,6 +8,10 @@ $data-red:    #ee0000;
 $data-yellow: #cbd114;
 $data-purple: #d51eff;
 $data-indigo: #6b00d2;
+$data-white: #ffffff;
+$data-peach: #f1d9ae;
+$data-gray: #a5a5a5;
+$data-brown: #915e3b;
```

Comment:
We should make a note to test the new colors in the Graph (XY Plot) tile too.  I suspect they will get automatically picked up there.

Reply:
I started to add some basic color checks for the XY Plot in the latest commit. I thought it would be easier, but I got a little tripped up in the directory structure and setting things up to be shared between the Bar graph and XY Plot. Do those changes seem OK? Or should I roll them back and just add a comment somewhere about adding color checks to the XY Plot in the future?